examples/browser/package.json

```diff
@@ -94,14 +94,16 @@
     "@theia/userstorage": "1.66.0",
     "@theia/variable-resolver": "1.66.0",
     "@theia/vsx-registry": "1.66.0",
-    "@theia/workspace": "1.66.0"
+    "@theia/workspace": "1.66.0",
+    "copyfiles": "^2.4.1"
   },
   "scripts": {
     "clean": "theia clean",
     "build": "theiaext build && npm run -s bundle",
     "build:production": "theiaext build && npm run -s bundle:production",
     "bundle": "npm run rebuild && theia build --mode development",
     "bundle:production": "npm run rebuild && theia build --mode production",
+    "postbuild": "copyfiles -u 6 '../../packages/terminal/src/node/shell-integrations/**' './lib/backend/' && copyfiles -u 6 '../../packages/terminal/src/node/shell-integrations/**/.*' './lib/backend/'",
```

Comment:
We should not add a `postbuild` step here as it will only work for the Browser Example application.

If this is a step all adopters need to perform too, then we should integrate it into the generated webpack configuration instead.